src/java.base/share/classes/jdk/internal/loader/BuiltinClassLoader.java

```diff
@@ -636,30 +617,20 @@ private List<URL> findMiscResource(String name) throws IOException {
         }
 
         // search all modules for the resource
-        List<URL> urls;
-        try {
-            urls = AccessController.doPrivileged(
-                new PrivilegedExceptionAction<>() {
-                    @Override
-                    public List<URL> run() throws IOException {
-                        List<URL> result = null;
-                        for (ModuleReference mref : nameToModule.values()) {
-                            URI u = moduleReaderFor(mref).find(name).orElse(null);
-                            if (u != null) {
-                                try {
-                                    if (result == null)
-                                        result = new ArrayList<>();
-                                    result.add(u.toURL());
-                                } catch (MalformedURLException |
-                                         IllegalArgumentException e) {
-                                }
-                            }
-                        }
-                        return (result != null) ? result : Collections.emptyList();
-                    }
-                });
-        } catch (PrivilegedActionException pae) {
-            throw (IOException) pae.getCause();
+        List<URL> urls = null;
+        for (ModuleReference mref : nameToModule.values()) {
+            URI u = moduleReaderFor(mref).find(name).orElse(null);
+            if (u != null) {
+                try {
+                    if (urls == null)
+                        urls = new ArrayList<>();
+                    urls.add(u.toURL());
+                } catch (MalformedURLException | IllegalArgumentException e) {
+                }
+            }
+        }
+        if (urls == null) {
+            urls = List.of();
         }
```

Comment:
Please remove lines 639-641: `urls == null` could never be true after this (they appear to be left over from a previous less-than-ideal merge).

Reply:
Removed